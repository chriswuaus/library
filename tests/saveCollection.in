SAVE COLLECTION Yeehaw.txt
add collection test.csv
save collection yeehaw.txt
exit
