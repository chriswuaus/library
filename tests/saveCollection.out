user: No books in system.

user: 17 books successfully added.

user: Success.

user: Ending Library process.
